helm/values.yaml

```diff
@@ -31,6 +31,18 @@ serviceAccount:
   # If not set and create is true, a name is generated using the fullname template
   name: ""
 
+rbac:
+  # Specifies how to manage access for kro in the cluster
+  # Two modes are currently supported:
+  # - unrestricted: Grants cluster-wide full access to all resources
+  # - aggregation: Grants a minimal set of permissions for kro to manage its own
+  #                resources, and additionally aggregates ClusterRoles with the
+  #                label `rbac.kro.run/aggregate-to-controller: "true"` to allow
+  #                kro users to extend kro's permissions.
+  # Recommended setting is aggregation, but unrestricted remains the default for
+  # backwards compatibility.
+  mode: unrestricted
```

Comment:
Love this, i'm thinking in the future we can add another option `impersonation-only` that only kro permissions to impersonate other service accounts and delegates that to admins.